docker-compose.prod.yml

```diff
@@ -10,25 +10,12 @@ services:
       target: runtimeimg
     image: patchman-engine_patchimg
 
-  db_volumefix:
-    container_name: db_volumefix
-    build:
-      context: .
-      dockerfile: dev/database/Dockerfile
-    image: patchman-engine_db
```

Comment:
I think  `docker-compose.prod.yml` is not consistent. The image "patchman-engine_db" build was removed but it's still referenced `db` container. Same for docker-compose.yml

Reply:
Yep, build part have to be moved to db container.